src/screens/Portfolio/Greetings.js

```diff
@@ -21,34 +22,30 @@ const getCurrentGreetings = () => {
   return "portfolio.greeting.morning";
 };
 
-class Greetings extends PureComponent<{
-  nbAccounts: number,
-}> {
-  render() {
-    const { nbAccounts } = this.props;
-    return (
-      <View style={styles.root}>
-        <LText secondary color="darkBlue" style={styles.title} bold>
-          {<Trans i18nKey={getCurrentGreetings()} />}
-        </LText>
-        <LText
-          secondary
-          color="grey"
-          style={styles.description}
-          numberOfLines={2}
-        >
-          {
-            <Trans
-              i18nKey="portfolio.summary"
-              count={nbAccounts}
-              values={{ count: nbAccounts }}
-            />
-          }
-        </LText>
-      </View>
-    );
-  }
-}
+const Greetings = ({ nbAccounts }: { nbAccounts: number }) => {
+  const { t } = useTranslation();
+  return (
+    <View style={styles.root}>
+      <LText secondary color="darkBlue" style={styles.title} bold>
+        {t(getCurrentGreetings())}
+      </LText>
+      <LText
+        secondary
+        color="grey"
+        style={styles.description}
+        numberOfLines={2}
+      >
+        {
+          <Trans
+            i18nKey="portfolio.summary"
+            count={nbAccounts}
+            values={{ count: nbAccounts }}
+          />
```

Comment:
Maybe switch completely to the i18n useTranslation hook rather than having 2 translation methods in the comp ?

Reply:
Indeed, wasn't sure how to handle plurals (if possible) with `t` function but it looks like it's quite easy: https://www.i18next.com/translation-function/plurals 👌